projects/kubernetes/boot.sh

```diff
@@ -58,7 +58,11 @@ elif [ $# -gt 1 ] || [ $# -eq 1 -a -n "${KUBE_PRESERVE_STATE}" ] ; then
     img="kube-node"
     name="node-${1}"
     shift
-    data="{\"kubeadm\": {\"join\": \"${*}\"} }"
+
+    if [ $# -ge 1 ] ; then
+    	data="{\"kubeadm\": {\"join\": \"${*}\"} }"
+    fi
```

Comment:
Took me a moment to get my head around the inversion of the logic here, but I think it's correct.